app/views/manuals/_content_for_manual.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :extra_headers do %>
-  <% if presented_manual.summary.present? %>
-    <meta name='description' content='<%= presented_manual.summary %>' />
+  <% if description.present? %>
+    <meta name='description' content='<%= description %>' />
```

Comment:
Now you've done this, you can use this `content_for_manual` partial in the `updates.html.erb` and `updates.html+new_navigation.erb` templates too.